web/help/codeworld.md

```diff
@@ -1406,6 +1409,17 @@ Interactions
 
 TODO: Write this section.
 
+This section will discuss using the interactionOf function in place of the simulationOf function,
```

Comment:
When you write inline code, you can typeset it in a fixed-width font by using backquotes (left of the 1 key).  It would be good to do that for `interactionOf` and `simulationOf` here, and also in the next change below.  It emphasizes that these are words known to the computer.